persistence/docs/CHANGELOG.md

```diff
@@ -7,6 +7,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```

Comment:
Can you add a TODO to make sure we consolidate `persistence/docs/CHANGELOG` and `persistence/CHANGELOG.md`